generators/arduino/oxocard.js

```diff
@@ -19,20 +19,23 @@ Blockly.Arduino.oxocard_turn_off = function(block) {
 };
 
 Blockly.Arduino['oxocard_turn_off_with_buttons'] = function(block) {
-	var valueL1= this.getFieldValue("L1").toLowerCase();
-	var valueL2= this.getFieldValue("L2").toLowerCase();
-	var valueL3= this.getFieldValue("L3").toLowerCase();
-	var valueR1= this.getFieldValue("R1").toLowerCase();
-	var valueR2= this.getFieldValue("R2").toLowerCase();
-	var valueR3= this.getFieldValue("R3").toLowerCase();
-	return 'oxocard.system->turnOff(oxocard.button->createButtonByte('
-		+valueL1 +', ' +valueL2 +', ' + valueL3 +', ' +valueR1 +', '
-		+valueR2 +', ' +valueR3 +'));\n';
-};
-
-Blockly.Arduino.oxocard_handle_autoturnoff = function() {
+	console.log(this);//blup
```

Comment:
I can find this "//blup" everywhere in your codebase.
what does it mean? why do you do it? maybe stop it?

also please, remove any console.log from you're code, that obviously was used for debugging, but is not an actual log-message.